app/components/BloodTypeDataTile.tsx

```diff
@@ -1,29 +1,61 @@
-import clsx from 'clsx'
+import clsx from "clsx";
 
-import DataChange from "./DataChange"
+import DataChange from "./DataChange";
+import { Popover, PopoverButton, PopoverPanel } from "@headlessui/react";
+import { regionStates } from "~/data/states";
+import InformationCircle from "~/icons/information-circle";
+import { TimeBreakdown } from "~/utils";
 
 type Props = {
-  label: string
-  count: number
-  change: number
-}
+  label: string;
+  count: number;
+  change: number;
+  waitTimes: Array<{ wait_list_time: string; count: number }>;
+};
 
-export default function BloodTypeDataTile({ label, count, change }: Props) {
+export default function BloodTypeDataTile({
+  label,
+  count,
+  change,
+  waitTimes,
+}: Props) {
   return (
     <div className="flex flex-col justify-center items-center">
       <span className="font-semibold">{label}</span>
       {/* Tile */}
-      <div className={
-        clsx(
-          change > 0 && 'border-red-500 bg-red-50',
-          change < 0 && 'border-green-500 bg-green-50',
-          'flex flex-col justify-center items-center border rounded w-16 h-16 shadow-sm relative'
-        )}>
-        <label htmlFor="">{count}</label>
-        <div className="absolute bottom-1">
-          <DataChange count={change} />
-        </div>
-      </div>
+      <Popover className="relative">
+        <PopoverButton
+          className="flex items-center"
+          aria-label={`Show waiting time breakdown`}
+        >
+          <div
+            className={clsx(
+              change > 0 && "border-red-500 bg-red-50",
+              change < 0 && "border-green-500 bg-green-50",
+              "flex flex-col justify-center items-center border rounded w-16 h-16 shadow-sm relative"
+            )}
+          >
+            <label htmlFor="">{count}</label>
+            <div className="absolute bottom-1">
+              <DataChange count={change} />
+            </div>
+          </div>
+        </PopoverButton>
+        <PopoverPanel
+          anchor={{ to: "bottom start", gap: "4px" }}
+          modal
+          focus
+          className="flex flex-col bg-white border rounded p-2"
+        >
+          <ul>
+            {waitTimes.map((d, index) => (
+              <li key={index}>
```

Comment:
purely fyi:
another way to make these key value type info sections with semantic html:
``` html
<dl>
    <dt>Key1</dt>
    <dd>Value1</dd>
    <dt>Key2</dt>
    <dd>Value2</dd>
</dl>
```

I think also if you made the counts bold, or vice versa it could help with making the numbers easily scannable, but other than that looks good.